gradle/libraries.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # The version we use by default in tests and benchmarks
 # Can be as high as IJ supports
-android-plugin = "8.2.2"
+android-plugin = "8.7.0"
```

Comment:
Is this required to build with Kotlin 2.1.0? If yes, is there an AGP <-> KGP compatibility matrix somewhere?

Reply:
👍 I'm all for more granular releases

Re: build-time AGP, I'm team "update to the latest version that's working in IJ" but I can never remember which one it is 🙃

Reply:
Ah there is something [here](https://kotlinlang.org/docs/multiplatform-compatibility-guide.html#version-compatibility) actually. It says AGP 8.2.2 should be compatible with KGP 2.1.0 👍